R/wcvp_checklist.R

```diff
@@ -85,15 +86,17 @@ wcvp_checklist <- function(taxon = NULL, taxon_rank = c("species", "genus", "fam
 
   if (is.null(wcvp_distributions)) {
     wcvp_distributions <- rWCVPdata::wcvp_distributions
-  }
+    wcvp_version <- rWCVPdata::wcvp_version()
+  } else wcvp_version <- "Locally provided copy of WCVP"
```

Comment:
Hahah using braces for the if but not for the else always looks like it will break to me - happy to go with your edit and push past my irrational fear